test/ContainerCommandLoaderTest.php

```diff
@@ -10,7 +10,7 @@
 
 namespace LaminasTest\Cli;
 
-use Laminas\Cli\ContainerCommandLoader;
+use laminas\cli\containercommandloader;
```

Comment:
Maybe I missed something but is this change correct?

Reply:
It is not correct. It is caused by CS rule, as this class is "virtual" (created by class_alias), and PHP stores these aliases always as lowercase, that's why the sniff doesn't work correctly here. I couldn't find a way to detect that it is an alias and skip the CS sniff on these...

Reply:
Is this something we can or should change in the CS? Or just ignore it here. Maybe we can at least mention it in the docs.

Reply:
I figured it out; I'll be pushing an update soon, once I resolve the other feedback. :smile: